engine/src/main/java/org/terasology/rendering/dag/nodes/FinalPostProcessingNode.java

```diff
@@ -203,6 +187,6 @@ public void propertyChange(PropertyChangeEvent event) {
             }
         } // else: no other cases are possible - see subscribe operations in initialize().
 
-        worldRenderer.requestTaskListRefresh();
+        // worldRenderer.requestTaskListRefresh();
```

Comment:
I'm sorry, this was an oversight (I was trying to track down a nasty bug, and was trying to isolate the problem by disabling the new parts of code I had added).

Enabled it again.